src/main/java/org/apache/sling/feature/extension/content/ContentHandler.java

```diff
@@ -90,20 +92,6 @@ private static ExecutionPlanBuilder buildExecutionPlan(Collection<Artifact> arti
 
     }
 
-    private static void extractSubPackages(FSPackageRegistry registry, ExecutionPlanBuilder builder, PackageId pid)
-            throws IOException {
-        Map<PackageId, SubPackageHandling.Option> subPkgs = registry.getInstallState(pid).getSubPackages();
```

Comment:
This is no longer possible as the getInstallState is a private method. IMHO this shouldn't be necessary at all, as during extraction all subpackages are automatically extracted. No need to flatten those before...
@DominikSuess What was the reason why you extracted the sub packages eagerly?